lib/ui_layer/widgets/search_tab/search_result.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:studentmap/domain_layer/building_manager/building.dart';
+
+class SearchResult extends StatelessWidget {
+  const SearchResult(this.building, {super.key});
+
+  final Building building;
+
+  @override
+  Widget build(BuildContext context) => Expanded(
+          child: GestureDetector(
+        onTap: () async {
+          // TODO: Navigate to the position on the map
+        },
+        child: Text(
+          "   ${building.name}",
```

Comment:
We should probably wrap Text inside of a Padding e.g. `Padding(padding: EdgeInsets.only(left: 4), child: Text())` instead of padding the string with spaces

Reply:
I've also fixed this in [SOF-74](https://github.com/softengsquad/studentmap/pull/38)